### ISSUE TITLE
Can you write a quick tutorial to run this on Google Colab?

### ISSUE THREAD

**Issue by nileshlg2003** [chrispolo/Mobilenet-_v1-Mask-RCNN-for-detection#1]
Hi,

Thanks for awesome work!
I want to use Mask RCNN for transfer learning and then using in mobile app. However, first, I would like to set it up in Google colab. Can you if possible, write up a quick tutorial for google colab and integration with mobile?

**Comment by chrispolo** [comment]
@nileshlg2003 Sorry for replying so late.
If you want to use mask rcnn  in the mobile devices and set it up in Google colab.
First, you need to apply for a Colaboratory and then upload the project includes .ipynb/.py/.h5/images
Then,you need to  set up the verion to python and choose to  use the GPU .
and now if you  want to run the demo, you need to install  keras/tensorflow/Numpy, skimage, scipy, Pillow, cython, h5py packages.

**Comment by nileshlg2003** [comment]
Thanks, I will try it
